sdk/template/azure-sdk-template/src/main/java/com/azure/sdk/template/Hello.java

```diff
@@ -1,6 +1,6 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
-package com.microsoft.azure.template;
+package com.azure.sdk.template;
```

Comment:
It also seems to be following the old naming for directory structure.
Should we consider updating that too?
![image](https://user-images.githubusercontent.com/16845631/88238694-2f79cb00-cc37-11ea-9d37-82a8b9119e94.png)

Reply:
The directories are updated too. `src/main/java/com/azure/sdk/template/Hello.java`